values.yaml

```diff
@@ -196,12 +199,13 @@ secrets:
       ofUOhwAXuMsTtuLagOMyAJVs+KRVrvnXGT/p9l213ZAnDtFSpkvcjD9WUcupeTca
       BjdoJBzImjVB5znOgIui3ME5
       -----END PRIVATE KEY-----
-
 ##
 ## StackStorm HA Cluster pod settings for each individual service/component.
 ##
 # Many st2web instances, placed behind a load balancer that serve web app and proxify requests to st2auth, st2api, st2stream.
 st2web:
+  protocol:
+    http: false
```

Comment:
As for `st2web` HTTP/HTTPS.

I agree that the feature makes sense too, but let's please split it into a separated PR as that's a different code and discussion.

A quick story there: we thought about exposing `st2web` via HTTP by default and offload HTTPS/certificates to optional Service Type like Ingress Controller #6 or LoadBalancer settings, but not sure, might need more input.
For reference, how other Helm roles usually do this: https://github.com/helm/charts/tree/master/stable/kibana#configuration https://github.com/helm/charts/tree/master/stable/hubot#configuration